prometheus/config/prometheus.yml

```diff
@@ -9,3 +9,13 @@ scrape_configs:
   - job_name: grafana
     static_configs:
       - targets: ['grafana:3000']
+  - job_name: cht-express-metrics
+    metrics_path: /api/v1/express-metrics
+    file_sd_configs:
```

Comment:
@jkuester I'm struggling a bit with this use of `file_sd_configs`.   Seems mandatory that this would re-use the same cht-instances.yml file as the json_exporter.

The [example cht-instances.yml file](https://github.com/medic/cht-watchdog/blob/main/development/fake-cht/example-config/cht-instances.yml#L4) includes the protocol (eg. https). I'm seeing errors for this new job saying the host is invalid.

From [docs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#file_sd_config), I'm seeing that the yml should contain a list of hosts. Online I'm seeing that these files don't include the protocol (https).

This works when I have cht-instances.yml like

```
- targets:
    - gamma.dev.medicmobile.org
```

Reply:
I believe we landed on including the protocol in the cht-instances.yml file because the https://github.com/prometheus-community/json_exporter seems to expect the protocol to be set for the provided `targets`. We considered just re-labeling to include the protocol when setting the targets, but that would make it tricky to scrape from `http` endpoints (e.g. if someone is just trying it out locally).

So, I think you are correct that re-labeling is the proper approach here.  I have not tested this, but since you are already stripping all the protocol info for the `instance` label, I think something like this might work:

```
      - source_labels: [instance]
        target_label: __address__
```

I don't think so, but you might also need to include:
```
        regex: '.*'
        replacement: "$1"
```

Reply:
Yep - first one works